src/python/pants/backend/python/providers/python_build_standalone/scripts/generate_urls.py

```diff
@@ -0,0 +1,98 @@
+# Copyright 2023 Pants project contributors (see CONTRIBUTORS.md).
```

Comment:
I have successfully run this script.

Reply:
Can you add a description to `docs/docs/contributions/development/maintenance-tasks-and-scripts.mdx`?

This also doesn't need to block merging.

(I also personally like using `argparse` with at least a one-sentence description, even if there's no args, so that `... --help` works and does something useful, but that's not so necessary.)